app/services/api/v1/ecf_participants/index.rb

```diff
@@ -74,47 +69,6 @@ def updated_since
         rescue ArgumentError
           Time.iso8601(URI.decode_www_form_component(filter[:updated_since]))
         end
-
-        # inner most query
-        # this query deals with the following scenario
-        # given one profile with 2 induction records
-        # we only want to return the latest induction record
-        # we also exclude any partnerships that have been challenged
-        def induction_record_ids_with_deduped_induction_records
-          query = InductionRecord
-            .joins(induction_programme: { partnership: %i[lead_provider cohort] })
-            .joins(participant_profile: %i[user participant_identity])
-            .select("DISTINCT ON (participant_profiles.id) participant_profile_id, induction_records.id")
-            .where(
-              induction_programme: {
-                partnerships: {
-                  challenged_at: nil,
-                  challenge_reason: nil,
-                  lead_provider:,
-                  cohort: with_cohorts,
-                },
-              },
-            )
-            .order("participant_profiles.id", start_date: :desc)
-            .to_sql
-
-          ActiveRecord::Base.connection.query_values("SELECT id FROM (#{query}) AS inner_query")
-        end
-
-        # second inner most query
-        # this query deals with where a user can have multiple profiles
-        # the work to map one user to one profile has not been done
-        # therefore we must work around and select correct profile
-        def induction_record_ids_with_deduped_profiles
```

Comment:
we have lost this de-dupe. therefore it should be noted that if a user has more than one profile:
- before this change there would be one record returned for a user
- after this change we now show as many records as they have profiles